scripts/setup

```diff
@@ -46,8 +46,7 @@ fi
 ####
 setup_header "Installing requirements..."
 
-install_nix && \
```

Comment:
At this point we might as well get rid of `scripts/lib/setup/installers.sh` and just put the contents of its `install_nix` method here. Make it simpler. Less files is better.

Reply:
Done.